.github/workflows/github_pages.yml

```diff
@@ -3,7 +3,7 @@ name: Github pages
 on:
   push:
     branches:
-      - releases/v1.0.0
+      - releases/**
```

Comment:
Why don't we build docs for `develop` change also?

Reply:
I applied the change from https://github.com/openvinotoolkit/datumaro/pull/881. @yunchu Can you check it?

Reply:
The develop only pipeline is planned to be added in the future, right?

Reply:
yes, right. and that will generate docs and exposed as 'latest' through the different URL from the release one (stable).